PicaSWB.js

```diff
@@ -39,7 +48,12 @@ var nachnameMapping = {
 };
 var nameMapping = {
 	"Berners-Lee, Tim" : "!18195804X!" // http://swb.bsz-bw.de/DB=2.1/PPNSET?PPN=18195804X&INDEXSET=1
-}
+};
+var languageMapping = {
```

Comment:
@zuphilip languageMapping entsprechend der in Pica3 gütlichen Abkürzungsregel umzusetzen http://swbtools.bsz-bw.de/winibwhelp/Liste_1500.htm 
"en" : "eng",
"de" : "ger",
"fr" : "fre", 
etc.